### ISSUE TITLE
nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.

### ISSUE THREAD

**Issue by JasonChenhx** [milesial/Pytorch-UNet#75]
My project encoutered this warning, although I can see the result, the effect is poor. So how can I modify my code to use this nn.functional.sigmoid .
Thanks

**Comment by milesial** [comment]
I don't think using torch.sigmoid instead of nn.functional.sigmoid will improve your results, as they are both the sigmoid function. You can submit a PR to get rid of the warning by using the newest function if you wish.

**Comment by E3V3A** [comment]
@milesial 
Why was this issue closed when it hasn't been fixed yet?

**Comment by sarra-rgb** [comment]
> I don't think using torch.sigmoid instead of nn.functional.sigmoid will improve your results, as they are both the sigmoid function. You can submit a PR to get rid of the warning by using the newest function if you wish.

so how can i solve this problem please ?

**Comment by milesial** [comment]
@sarra-rgb How is it a problem ? Open a PR if you want to get rid of the warning.
